invenio_app_ils/acquisition/config.py

```diff
@@ -136,7 +136,10 @@
     ),
     acq_vendors=dict(  # VendorSearch.Meta.index
         name=dict(
-            fields=["name"], title="Name", default_order="desc", order=1
+            fields=["name.keyword"],
```

Comment:
I am not sure I understand the fix: if I a vendor called `Amazon`, and I search for `Amaz`, I won't find it I believe... Shouldn't we search in the text instead?

Reply:
@ntarocco yes we should do the search in text, but then we encounter the following issue `Fielddata is disabled on text fields by default.`, it is described in detail in 
https://www.elastic.co/guide/en/elasticsearch/reference/current/fielddata.html